release-notes/README.md

```diff
@@ -4,6 +4,16 @@ description: Release notes and change logs for the Adobe Experience Platform Mob
 
 # Release notes
 
+## May 9, 2022
+
+### iOS AEPCore 3.6.0
+
+* App Extension Support
+* Add a new API to the Extension protocol for getting latest non pending shared state
+* Bundled Rules support
```

Comment:
Do we have a bit more information on this?

Reply:
Going to be updating public doc with more information on this next. As well as for the shared state api. But i don't really know what else to say here without getting too detailed. It's like [this feature](https://aep-sdks.gitbook.io/docs/foundation-extensions/mobile-core/configuration#using-a-bundled-file-configuration) but for rules.

Reply:
Ahh, got it. Maybe add a link to the updates for Bundled Rules for further clarity?